tests/e2e/functional_test.go

```diff
@@ -579,8 +579,14 @@ var _ = Describe("Functional e2e", Serial, func() {
 			if isbCondStatus == metav1.ConditionTrue || plCondStatus == metav1.ConditionTrue {
 				return false
 			}
-			if isbRollout.Status.UpgradeInProgress != apiv1.UpgradeStrategyNoOp || plRollout.Status.UpgradeInProgress != apiv1.UpgradeStrategyNoOp {
-				return false
+			if upgradeStrategy == "progressive" {
+				if isbRollout.Status.UpgradeInProgress != apiv1.UpgradeStrategyProgressive || plRollout.Status.UpgradeInProgress != apiv1.UpgradeStrategyNoOp {
+					return false
+				}
+			} else {
+				if isbRollout.Status.UpgradeInProgress != apiv1.UpgradeStrategyNoOp || plRollout.Status.UpgradeInProgress != apiv1.UpgradeStrategyNoOp {
+					return false
+				}
```

Comment:
maybe it could improve readability if we remove this entire block of code that you've changed here, and instead make it so we only do this entire "Verify that dependent Pipeline is not paused when an update to ISBService not requiring pause is made" section if we're doing PPND? 

I think this test doesn't make sense if we're using another strategy.